misc/python/materialize/cli/ci_logged_errors_detect.py

```diff
@@ -113,6 +113,8 @@
     | internal\ error:\ no\ AWS\ external\ ID\ prefix\ configured
     # For persist-catalog-migration ignore failpoint panics
     | persist-catalog-migration-materialized.* \| .* failpoint\ .* panic
+    # For tests we purposely trigger this error
+    | persist-catalog-migration-materialized.* \| .* incompatible persist version \d+\.\d+\.\d+(-dev)?, current: \d+\.\d+\.\d+(-dev)?, make sure to upgrade the catalog one version at a time
```

Comment:
This doesn't seem to be working and the test is still failing due to this log, any advice?